sysbrokers/IB/ib_config_futures.csv

```diff
@@ -13,7 +13,10 @@ AUD-SGX,AU,SGX,USD,25000,1,FALSE
 AUDCAD,ACD,CME,CAD,200000,1,FALSE
 AUDJPY,AJY,CME,JPY,200000,1,FALSE
 AUDJPY-SGX,AJ,SGX,JPY,25000,1,FALSE
+AUS10,XT,SNFE,AUD,10000,1,FALSE
+AUS3,YT,SNFE,AUD,1000,1,FALSE
```

Comment:
No please dont do this - I like the IB and instrument configs to match

Reply:
Sure, I'll remove them again. I just put them back because I noticed that you hadn't deleted them from the ib_config_futures and I wanted to be consistent with what you did.

Reply:
Yes I forgot... wasn't deliberate.